app/views/administrate/application/_search.html.erb

```diff
@@ -1,25 +1,22 @@
 <form class="search" role="search">
-  <span class="search__icon">
-    <svg class="search__clear-icon">
+  <label class="search__label" for="search">
+    <svg class="search__eyeglass-icon" role="img">
+      <title>Search <%= resource_name %></title>
       <use xlink:href="#icon-eyeglass" />
     </svg>
-  </span>
-  <input
-    type="text"
-    name="search"
-    class="search__input"
-    placeholder="Search"
-    value="<%= search_term %>"
-    aria-label="Search"
-  />
-  <span class="search__hint">
-    Press enter to search
-  </span>
-  <span class="search__clear">
-    <%= link_to clear_search_params do %>
-      <svg class="search__clear-icon">
-        <use xlink:href="#icon-cancel" />
-      </svg>
-    <% end %>
-  </span>
+  </label>
+
+  <input class="search__input"
+         id="search"
+         type="search"
+         name="search"
+         placeholder="Search <%= resource_name %>"
+         value="<%= search_term %>">
```

Comment:
It appears to work just fine…

FWIW, this PR doesn’t change this `value`, so if we haven’t had any bug reports, I think we’re good.